.github/workflows/community.yml

```diff
@@ -1,7 +1,7 @@
 on:
   fork:
   push:
-    branches: [ main ]
+    branches: [ develop ]
```

Comment:
This branch does not exist on this repo. I do not think we need a `develop` branch on this repo as there is not much activity